### PR TITLE
Amend worker reboot to use fly

### PR DESCRIPTION
**Pull request by iw-gtadam**

### CODE REVIEW

File: ci/jobs/reboot-worker-management.yml.j2 [hunk 1]
```diff
@@ -7,4 +7,12 @@ jobs:
     config:
       params:
         AWS_ACC: "{{accounts['management']}}"
+  - .: (( inject meta.plan.fetch-credentials ))
+    config:
+      params:
+        USERNAME_KEY: "/ci/0df691ef-5c06-e638-ae85-d14bb1e0703f-user"
+        PASSWORD_KEY: "/ci/0df691ef-5c06-e638-ae85-d14bb1e0703f-password"
```
Comment (ChrisScottThomas) [inline ci/jobs/reboot-worker-management.yml.j2:14]: Why are these here if we're doing `fetch-credentials` ?

Comment (iw-gtadam) [reply ci/jobs/reboot-worker-management.yml.j2:14]: They are the keys that refer to the username and password values

Comment (iw-gtadam) [reply ci/jobs/reboot-worker-management.yml.j2:14]: My bad will remove

Comment (ChrisScottThomas) [reply ci/jobs/reboot-worker-management.yml.j2:14]: Check with Jim B before merge

File: ci/jobs/reboot-worker-management-dev.yml.j2 [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    trigger: false
+  - .: (( inject meta.plan.create-aws-profiles ))
+    config:
+      params:
+        AWS_ACC: "{{accounts['management-dev']}}"
+  - .: (( inject meta.plan.fetch-credentials ))
+    config:
+      params:
+        USERNAME_KEY: "/ci/2f0968b1-63a9-a026-102c-36550c89e0ca-user"
+        PASSWORD_KEY: "/ci/2f0968b1-63a9-a026-102c-36550c89e0ca-password"
```
Comment (jimbarker) [inline ci/jobs/reboot-worker-management-dev.yml.j2:14]: We should never hardcode parameters in open-source repositories, not even as a temporary measure.

File: ci/jobs/reboot-worker-management.yml.j2 (hunk 1 above)
Comment (jimbarker) [inline ci/jobs/reboot-worker-management.yml.j2:14]: We should never hardcode parameters in open-source repositories, not even as a temporary measure.
